src/cdrizzlebox.c

```diff
@@ -657,18 +677,21 @@ do_kernel_turbo(struct driz_param_t* p) {
                 nxa = fortran_round(xxa);
                 nyi = fortran_round(yyi);
                 nya = fortran_round(yya);
-                iis = MAX(nxi, 0);  /* Needed to be set to 0 to avoid edge effects */
-                iie = MIN(nxa, osize[0]-1);
-                jjs = MAX(nyi, 0);  /* Needed to be set to 0 to avoid edge effects */
-                jje = MIN(nya, osize[1]-1);
+                iis = MAX(nxi,
```

Comment:
The comment is causing this odd formatting.  I recommend moving it to the line above, instead of at the end of the line.